### PR TITLE
code adjustments: use pipe operator

### PR DESCRIPTION
**Pull request by AndyGura**

### CODE REVIEW

File: src/TokenHandler.mo [hunk 1]
```diff
@@ -93,8 +92,7 @@ module TokenHandler {
 
     let size = Nat8.toNat(bytes[size_index]);
     if (size_index + size != 31) return null;
-
-    ?Principal.fromBlob(Blob.fromArray(Array.tabulate(size, func(i : Nat) : Nat8 = bytes[i + 1 + size_index])));
+    Array.tabulate(size, func(i : Nat) : Nat8 = bytes[i + 1 + size_index]) |> ?Principal.fromBlob(Blob.fromArray(_));
```
Comment (timohanke) [inline src/TokenHandler.mo:95]: Can we do 
```
|> Blob.fromArray(_)
|> Principal.fromBlob(_)
```
?

**Review by timohanke — COMMENTED** [review]
Left one comment

File: src/TokenHandler.mo (hunk 1 above)
Comment (AndyGura) [reply src/TokenHandler.mo:95]: Yes

Comment (AndyGura) [reply src/TokenHandler.mo]: Done

**Review by timohanke — APPROVED** [review]

**Pull request merged**